spec/unit/installer/user_project_integrator/target_integrator_spec.rb

```diff
@@ -301,19 +301,41 @@ module Pod
 
         it 'adds copy pods resources input and output paths' do
           resource_paths_by_config = {
-            'Debug' => ['${PODS_CONFIGURATION_BUILD_DIR}/DebugLib/DebugLib.bundle'],
-            'Release' => ['${PODS_CONFIGURATION_BUILD_DIR}/ReleaseLib/ReleaseLib.bundle'],
+            'Debug' => [
```

Comment:
can you expand this test case to include all of the possible extensions as specified in `output_extension_for_input_extension` method? I am sorry for being pedantic but I think its good to have test coverage for these things.

Reply:
If you look further, I _did_ add all possible extensions - some of them are in Debug, some in Release.

Reply:
you win. `.framework` is missing but meh.